### DISCUSSION TITLE
Some questions

### DISCUSSION THREAD

**Discussion by TAforever** [actionless/pikaur#841] [Q&A]
Recently installed pikaur and I like the program, but I have a few questions:
1. is it a bug or a feature that the font is black in color that it is almost invisible and can it be changed?
2. Is it possible to remove the warning that stable versions of some packages are "newer" than the ones under development?
![pikaur](https://github.com/user-attachments/assets/935a902a-70ab-4538-8f28-62be8f94e181)

**Comment by actionless** [comment]
1) it's a bug in your terminal colorscheme, if some color is not readable from the background

2) almost everything is "possible" 🤣

**Reply by TAforever** [reply]
1. In my example you can still see something, but if you take an unconfigured kitty or alacritty terminal with a black background, the font in this place will be invisible.
2. If this can be removed not by editing the source code, but by configuration, please tell me how. I read the manual and did not find this information there.

**Reply by TAforever** [reply]
![pikaur2](https://github.com/user-attachments/assets/b316545f-d75c-43d2-be80-52e703cf95ff)

**Reply by actionless** [reply]
1) you also can turn off the color in pikaur with cli flag, if you're not able to choose or generate a correct terminal colorscheme
2) a) https://github.com/actionless/pikaur/pulls
b) https://www.paypal.me/actionless

**Reply by actionless** [reply]
also re:1, i'm planing to make that easier for the people to generate such palettes, but it's just in a huge list of thing to do some day, until someone else would submit a patch earlier: https://github.com/themix-project/themix-gui/issues/337

**Reply by TAforever** [reply]
Good plan, but I think that for now it will be enough to change the color for "None" from black to neutral, because right now this inscription can be seen only if you have a light background in the terminal.
Anyway, thank you for your work
![pikaur3](https://github.com/user-attachments/assets/e9db3704-85fc-47a4-a381-2a29d5440324)

**Reply by actionless** [reply]
again, just choose/create a normal terminal palette. i see no reason to continue this conversation.
